### PR TITLE
Feature: SQLite query runner

### LINKED ISSUE #627: sqlite support
Is support for sqlite databases something feasible ?

### PR DESCRIPTION
**Pull request by gissehel**
This pull request add support for sqlite databases. 
No type support because the nature of sqlite's types makes it difficult to infer the best type for a query result.

Should close #627

### CODE REVIEW

File: redash/query_runner/sqlite.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+#from redash.query_runner import TYPE_DATETIME
+#from redash.query_runner import TYPE_BOOLEAN
+#from redash.query_runner import TYPE_INTEGER
+from redash.query_runner import TYPE_STRING
+#from redash.query_runner import TYPE_DATE
+#from redash.query_runner import TYPE_FLOAT
+#from redash.query_runner import SUPPORTED_COLUMN_TYPES
+from redash.query_runner import register
+#from redash.query_runner import get_query_runner
+#from redash.query_runner import import_query_runners
```
Comment (arikfr) [inline redash/query_runner/sqlite.py]: There is no point in keeping the commented out code... Let's remove it.

File: redash/query_runner/sqlite.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+class Sqlite(BaseQueryRunner):
+    @classmethod
+    def configuration_schema(cls):
+        return {
+            "type": "object",
+            "properties": {
+                "dbname": {
+                    "type": "string",
+                    "title": "Database Name"
```
Comment (arikfr) [inline redash/query_runner/sqlite.py]: Maybe `Database Path` will be more descriptive?

**Comment by arikfr** [comment]
:+1: looks good. I left two minor comments, let's sort those out and it can be merged.

**Comment by gissehel** [comment]
I corrected the points, but I still need to test it more.

**Comment by arikfr** [comment]
Merged. Thanks.

**Pull request merged**